api/XRPermissionStatus.json

```diff
@@ -1,102 +0,0 @@
-{
-  "api": {
-    "XRPermissionStatus": {
```

Comment:
I can't find this interface in Chromium, Gecko or WebKit except in tests/comments, so OK to remove.